### DISCUSSION TITLE
Restore pack (功能復原包)

### DISCUSSION THREAD

**Discussion by WhatTheBlock** [WhatTheBlock/WindowsSimplify#34] [General]
How to use？
1. Unzip restore pack.
2. Use powershell run the script `install.ps1`.
![Snipaste_2022-07-21_10-08-22](https://user-images.githubusercontent.com/34786954/180114401-9af7e18c-7073-4117-8e07-da63454b1f43.png)
3. Done.
----
If you have any questions about the restore pack, please ask in this discussion thread, thank you!

復原包有使用上的問題都請在這個討論串提問，謝謝！

**Comment by sean207cc** [comment]
hi，我想請問這是還原成正常win的還原包嗎?

**Reply by WhatTheBlock** [reply]
不是，只是依照需求安裝回商店或xbox之類的組件

**Comment by LYAN0616** [comment]
你好，有辦法安裝原本DEFEDER防毒嗎? 謝謝

**Reply by WhatTheBlock** [reply]
沒辦法，想要使用defender就別使用精簡系統了吧
